SimpleAnnPlayground/Debugging/FrmElementDesigner.Designer.cs

```diff
@@ -104,8 +104,8 @@ private void InitializeComponent()
             // 
             // PicDraw
             // 
+            this.PicDraw.Anchor = System.Windows.Forms.AnchorStyles.None;
             this.PicDraw.BackColor = System.Drawing.Color.White;
-            this.PicDraw.Dock = System.Windows.Forms.DockStyle.Fill;
```

Comment:
The `DockStyle` property for `PicDraw` should remain as `System.Windows.Forms.DockStyle.Fill`.

To get an updated value of the `PicDraw` center is necessary to handle [PicDraw_Resize()](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.control.resize?view=windowsdesktop-6.0) event.

Reply:
Oh, you're right, when I started I moved some options in the properties box, I didn't realise that I didn't get those back, not problem, I'll fix them. Thanks!